insights/parsers/mdadm.py

```diff
@@ -0,0 +1,74 @@
+"""
+MDAdm - command ``/usr/sbin/mdadm -E {device}``
+=================================================
+"""
+
+from insights.core import CommandParser
+from insights.core.plugins import parser
+from insights.parsers import split_kv_pairs
+from insights.parsers import ParseException
+
+from insights.specs import Specs
+
+
+@parser(Specs.mdadm_E)
+class MDAdmMetadata(CommandParser, dict):
+    """
+    Parser for output of ``mdadm -E`` for each MD device in system.
+
+    This stores the information from the output in the following properties:
+    * ``device`` - the name of the device after /dev/ - e.g. loop0
+
+    Sample output::
+    /dev/loop0:
```

Comment:
1. One more blank line is required for this `Sample output::` section.
2. One more indent block is required for the below content of the `Sample output::` section.